### PR TITLE
fix: updates helppanel tests and removes others

### PR DESCRIPTION
**Pull request by noahamac**
Thanks for adding that extension context code. I moved it over to a wrapper function we can reuse.

I ran into a whole bunch of trouble trying to get d3 to play nicely with jest and react-testing-library. I chose <HelpPanel /> to start working on because I thought jest would have access to the resultant svg, but that doesn't look like the case. 
https://github.com/jsdom/jsdom/issues/300

The answer I was able to find was to test the individual accessors. That doesn't make a lot of sense to me, as it's what Typescript is doing. And given my intention to eliminate d3 control of dom, doesn't seem efficient. 

I removed the snapshot tests because they weren't really testing anything as shallow renders. 

I can address by typing the svg. This is what I've proposed, and will work towards in medium term. In short term, the most important test targets are:
- Metadata panel and utils
- LookmlDiagrammer utils

As well as improving the integration tests and diagram generation tests, which I feel better about. 

I want to merge #47 , #49 , and this, check & update integration tests, and then cut a Marketplace release. Will work on the above items in future PRs, blocking net new work.

### CODE REVIEW

**Comment by noahamac** [comment]
Blocked by #49

File: .eslintcache [hunk 1]
```diff
@@ -1 +1 @@
...
...nge":"993","text":"910"},{"range":"993","text":"911"},{"range":"994","text":"910"},{"range":"994","text":"911"},{"range":"995","text":"910"},{"range":"995","text":"911"},[2126,2129],"unknown","never",[1970,1973],[1795,1798],[1835,1838],[2077,2080],[2564,2567],[2924,2927],[3391,3394],[1369,1372],[1516,1519],[1897,1900],[30,33],[27,30],[30,33],[27,30],[1798,1801],[2375,2378],[1661,1664],[1949,1952],[3953,3956],[4009,4012],[4081,4084],[4572,4575],[4628,4631],[4700,4703],[5798,5801],[1166,1169],[1186,1189],[1205,1208],[1237,1240],[1355,1358],[3683,3686],[2473,2476],[2591,2594],[4497,4500],[1524,1527],[1205,1208],[1705,1708],[2375,2378],[1148,1151],[1618,1621],[5197,5200],[5228,5231],[5236,5239],[5792,5795],[8120,8123],[8154,8157],[9896,9899],[9930,9933],[10724,10727],[10758,10761],[11529,11532],[11563,11566],[1482,1485],[1661,1664],[1724,1727],[7749,7752],[1198,1201],[1217,1220],[1224,1227],[1314,1317],[1355,1358],[1855,1858],[1867,1870],[1878,1881],[2028,2031],[4084,4087],[4617,4620],[4758,4761],[5360,5363],[5950,5953],[6613,6616],[7089,7092],[7150,7153],[7915,7918],[7962,7965],[8021,8024],[8250,8253],[8536,8539],[8567,8570],[8594,8597],[8625,8628],[8673,8676],[8706,8709],[2099,2102]]
```
Comment (bryans99) [inline .eslintcache:1]: git ignore this

File: src/__tests__/test-utils/render_with_extension.tsx [hunk 2]
```diff
@@ -1,58 +1,50 @@
...
-    expect(basic.debug()).toMatchSnapshot();
-  });
-});
+import React, { ReactElement } from 'react'
+import { renderWithTheme } from '@looker/components-test-utils'
+import {  ExtensionContext2, ExtensionContextData2 } from "@looker/extension-sdk-react"
+import {  LookerHostData,  registerHostApi, ExtensionHostApi } from "@looker/extension-sdk"
+import { Looker40SDK } from '@looker/sdk'
+
+const withExtensionContext = (
```
Comment (bryans99) [inline src/__tests__/test-utils/render_with_extension.tsx:27]: This is okay for now. should consider moving this as a test util to sdk codegen exported possibly from extension sdl react.

**Review by bryans99 — DISMISSED** [review]
I think we want to gitignore the eslintcache but I've mentioned this on another PR

File: .eslintcache (hunk 1 above)
Comment (noahamac) [reply .eslintcache:1]: done in #49

**Review by bryans99 — APPROVED** [review]

**Pull request merged**